src/server/scripts/Northrend/Naxxramas/boss_sapphiron.cpp

```diff
@@ -303,6 +306,7 @@ class boss_sapphiron : public CreatureScript
                             case EVENT_LIFTOFF:
                                 Talk(EMOTE_AIR_PHASE);
                                 me->SetDisableGravity(true);
+                                me->SetHover(true);
```

Comment:
I cannot confirm this without sniffs but from experience I can tell you that true hover is very rarely used - so this is likely to be incorrect